www/docs/src/components/page-title/page-title.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { ChevronRight } from 'lucide-react'
+import { type Props } from '@astrojs/starlight/props'
+
+const PageTitle = ({ entry }: Props) => {
+  const [crumbs, setCrumbs] = useState<string[]>([])
+  const { data } = entry
+
+  useEffect(() => {
+    const formattedCrumbs = window.location.pathname
+      .replace(/^\/|\/$/g, '')
+      .split('/')
+    setCrumbs(formattedCrumbs)
+  }, [])
+
+  const navigateTo = (crumb: string, _idx: number) => {
+    const constructedUrl = `/${crumb}/`
+    window.location.href = constructedUrl
+  }
+
+  return (
+    <div className="mt-8">
+      <div className="flex flex-row items-center gap-3 select-none">
+        {crumbs.map((crumb, idx) => (
+          <React.Fragment key={idx}>
+            <p
+              role="link"
+              className={`text-sm cursor-pointer ${idx !== crumbs.length - 1 || idx === 0 ? 'text-muted-foreground hover:text-foreground transition-all' : 'text-foreground'}`}
+              onClick={
+                idx !== crumbs.length - 1 ?
+                  () => navigateTo(crumb, idx)
+                : undefined
+              }>
+              {crumb}
+            </p>
+            {idx !== crumbs.length - 1 ?
+              <ChevronRight
+                className="text-muted-foreground"
+                size={16}
+              />
+            : null}
+          </React.Fragment>
+        ))}
+      </div>
+      <h1 className="text-2xl font-bold mt-8 mb-4">{data.title}</h1>
```

Comment:
From the [Starlight docs](https://starlight.astro.build/reference/overrides/#pagetitle):

> Implementations should ensure they set id="_top" on the <h1> element as in the default implementation.

```suggestion
      <h1 id="_top" className="text-2xl font-bold mt-8 mb-4">{data.title}</h1>
```